src/pages/NewGroupPage.vue

```diff
@@ -0,0 +1,132 @@
+<script lang="ts" setup>
+import { ref } from 'vue'
+import { useRouter } from 'vue-router'
+
+import { useGroupStore } from '/@/stores/group'
+import { useToastStore } from '/@/stores/toast'
+import { useUserStore } from '/@/stores/user'
+
+import type { Group, PostGroup } from '/@/lib/apis'
+import apis from '/@/lib/apis'
+
+import FormInput from '/@/components/shared/FormInput.vue'
+import FormInputNumber from '/@/components/shared/FormInputNumber.vue'
+import FormSelect from '/@/components/shared/FormSelect.vue'
+import FormTextarea from '/@/components/shared/FormTextarea.vue'
+import SimpleButton from '/@/components/shared/SimpleButton.vue'
+
+const router = useRouter()
+
+const userStore = useUserStore()
+const groupStore = useGroupStore()
+
+const toastStore = useToastStore()
+
+const group = ref({
```

Comment:
reactiveを使えば`.value`をいちいち書かなくてよかった気がします
```suggestion
const group = reactive({
```